.buildkite/pipeline-release.yml

```diff
@@ -0,0 +1,50 @@
+agents:
+  provider: "k8s"
+  image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:latest
+  memory: "4G"
+
+# for all merge in main or commit associated with a tag
+
+steps:
+
+  - group: release
+
+    steps:
+
+      - label: "k8s manifests"
+        if: |
+          ( build.branch == "main" && build.source != "schedule" )
+          || build.tag != null
+          || build.message =~ /^buildkite test .*release.*/
+        commands:
+          - buildkite-agent artifact download 'config/*.yaml' config/
+          - .buildkite/scripts/build/setenv.sh
+          - .buildkite/scripts/release/k8s-manifests.sh
+
+      - label: "helm release"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-release-helm.yml
+
+      - label: "redhat operatorhub"
+        if: |
+          build.message =~ /^buildkite test .*release.*/
+          || build.message == "release eck on operatorhub"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-release-operatorhub.yml
+
+      # - label: "redhat operatorhub"
```

Comment:
Should this be deleted?

Reply:
Yes, I was testing what it looks like to condensed all of pipeline-release-operatorhub.yml in one step. The advantage of spreading each command in different step is that we can do retry step per step.

Reply:
Yes I am deleting, I was testing what it looked like to condense the whole pipeline-release-operatorhub.yml into one step. Finally it's not really more readable and the advantage of splitting each command into different steps is that we can try again step by step.